pydis_site/apps/content/resources/guides/pydis-guides/contributing/bot.md

```diff
@@ -38,14 +37,101 @@ If your bot fails to start with a `PrivilegedIntentsRequired` exception, this in
 ### Configure the Bot
 You now have both the bot's code and a server to run it on. It's time you to connect the two by changing the bot's configurations.
 
-#### config.yml
-Entering the directory of the cloned code, you will find a file named `config-default.yml`.
+#### Scaffolding the configuration
+To make setup much easier, there is a file called `bootstrap_config.py` that represents a script to bootstrap the configuration for you and help you get started immediately
+without having to spend much time copying ids from your newly created server into your configuration file.
+
+**Note**
+
+This phase can be skipped and done manually, but would require extra manual work.
```

Comment:
Looking at the static deploy this currently looks ugly. This would be much better.

```
**Note**: This phase can be skipped and done manually, but would require extra manual work.
```